### PR TITLE
Modifycomment

### PR DESCRIPTION
**Pull request by xiaoyulei**
just update comment

### CODE REVIEW

File: cli/start.go [hunk 1]
```diff
@@ -175,7 +175,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	log.Flush()
 }
 
-// A exterminateCmd command shuts down the node server.
+// An exterminateCmd command shuts down the node server.
```
Comment (MBerhault) [inline cli/start.go]: you can remove "A/An" entirely, comments should start with the variable/function name.

**Comment by MBerhault** [comment]
LGTM

File: cli/start.go [hunk 2]
```diff
@@ -175,7 +175,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	log.Flush()
 }
 
-// A exterminateCmd command shuts down the node server.
+// exterminateCmd command shuts down the node server.
```
Comment (tamird) [inline cli/start.go]: this comment should mention that `exterminateCmd` also destroys all the data.

File: kv/dist_sender_test.go [hunk 3]
```diff
@@ -402,7 +402,7 @@ func TestRetryOnDescriptorLookupError(t *testing.T) {
 
 func TestEvictCacheOnError(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	// if rpcError is true, the first attempt gets an RPC error, otherwise
+	// if rpcError is true, the first attempt gets a RPC error, otherwise
```
Comment (tamird) [inline kv/dist_sender_test.go]: This should be `an`, I think.

**Comment by bdarnell** [comment]
LGTM

**Comment by YuleiXiao** [comment]
@tamird I think you are right. I will update it.

File: cli/start.go [hunk 4]
```diff
@@ -175,7 +175,8 @@ func runStart(cmd *cobra.Command, args []string) {
 	log.Flush()
 }
 
-// A exterminateCmd command shuts down the node server.
+// exterminateCmd command shuts down the node server and
+// destroy all data held by the node.
```
Comment (tamird) [inline cli/start.go]: destroys

**Comment by tamird** [comment]
@YuleiXiao can you squash this down to one commit?

**Comment by YuleiXiao** [comment]
@tamird you mean create a new PR with all?

**Comment by tamird** [comment]
@YuleiXiao `git rebase -i master` and then use `f` to fixup the commits

**Comment by YuleiXiao** [comment]
@tamird OK,I try it.

**Comment by YuleiXiao** [comment]
@tamird I had do what you told me. This is my first time to do like this. Is it correct please?

**Comment by tamird** [comment]
LGTM, thank you!

File: cli/start.go [hunk 5]
```diff
@@ -40,7 +40,7 @@ import (
 // Context is the CLI Context used for the server.
 var Context = server.NewContext()
 
-// An initCmd command initializes a new Cockroach cluster.
+// initCmd command initializes a new Cockroach cluster.
```
Comment (tschottdorf) [inline cli/start.go]: the word "command" seems superfluous in all instances now.

**Comment by YuleiXiao** [comment]
@tschottdorf yes. I can update it. But create another PR? I see this PR had already closed.

**Comment by tschottdorf** [comment]
don't worry about it, it's ok. Thanks

**Pull request merged**